### PR TITLE
Added path prefix `assign` to routes in `AssignController`

### PR DESCRIPTION
**Pull request by Corepex**
To fix the collision of route paths I added the a prefix (`assign`) to the path.

image

### CODE REVIEW

File: src/Tag/Controller/Element/AssignController.php [hunk 1]
```diff
@@ -48,10 +48,10 @@ public function __construct(
     /**
      * @throws ElementSavingFailedException|NotFoundException
      */
-    #[Route('/tags/{elementType}/{id}/{tagId}', name: 'pimcore_studio_api_assign_element_tag', methods: ['POST'])]
+    #[Route('/tags/assign/{elementType}/{id}/{tagId}', name: 'pimcore_studio_api_assign_element_tag', methods: ['POST'])]
```
Comment (martineiber) [inline src/Tag/Controller/Element/AssignController.php:51]: Sonar is now complaining that the line is too long :D

Pls split it up in multiple lines..

Comment (martineiber) [inline src/Tag/Controller/Element/AssignController.php]: Sonar is now complaining that the line is too long :D

Pls split it up in multiple lines..

**Review by martineiber — CHANGES_REQUESTED** [review]

File: src/Tag/Controller/Element/AssignController.php (hunk 1 above)
Comment (Corepex) [reply src/Tag/Controller/Element/AssignController.php]: done :)

**Review by martineiber — APPROVED** [review]
LGTM :+1:

**Pull request merged**